legit.py

```diff
@@ -1,54 +1,87 @@
 from git import Repo
 from git.exc import GitCommandError
 from typing import Iterable
+import logging
 
+# ==============================================================
+# create logger with 'legit'
```

Comment:
Code speaks better what he do here. So it looks that we do not need this comment here.